penney/__main__.py

```diff
@@ -1,12 +1,15 @@
 # -*- coding: utf-8 -*-
 """Penney main."""
+import sys
 from penney.functions import *
 
 if __name__ == "__main__":
-    while(True):
-        description()
-        menu()
-        control_input = input(
-            "\nPress [R] to restart or any other key to exit.")
-        if control_input.upper() != "R":
-            break
+    args = sys.argv
```

Comment:
Why is this for?
And shouldn't we add this change for `CHANGELOG.md`?

Reply:
It's a minor change in order to test `main` script.